bazel_tools/haskell-stack-snapshot-deps.patch

```diff
@@ -0,0 +1,144 @@
+From 14272ac578adbbfd1737f9f191951c2a49856120 Mon Sep 17 00:00:00 2001
```

Comment:
Also looks ustreamable?

Reply:
Yes, there's also a PR for this one. It's linked here https://github.com/digital-asset/daml/pull/2743/files#diff-0d048ba7b51a63feefd2338106bd6d99R51